internal/roundtrip.go

```diff
@@ -13,30 +13,45 @@ import (
 	"github.com/vmware-tanzu/observability-event-resource/wavefront"
 )
 
+type request struct {
+	method        string
+	token         string
+	response      string
+	requestString string
+}
+
 type fakeRoundTripper struct {
```

Comment:
This looks like test-only code, so I'd put it in a `_test.go` file to ensure it's not linked into an actual build.